docs/src/build.md

```diff
@@ -138,15 +139,15 @@ We need to install Ninja which can be done with the following command
 sudo apt install ninja-build
 ```
 
-### Ubuntu 20.04
+### Ubuntu 22.04
```

Comment:
Above, on line 133/134, there's already a `### Ubuntu 22.04` header, I guess these should be unified.

Reply:
Good catch, I merged the two! I just removed the Ubuntu 20 section, technically it supports compiler only build, but since we're not testing it folks might be on their own a bit.